### ISSUE TITLE
Public comment from Gartner Inc.

### ISSUE THREAD

**Issue by OMBPublicComments** [ombegov/cloud.cio.gov#26]
Apply Market Research Practices to the Cloud Smart Strategy

Gartner Inc. (Gartner) welcomes the opportunity to comment on the draft Federal Cloud Computing Strategy, “Cloud Smart”. Gartner is a leading provider of information technology research and consulting services to both the public and private sectors, and has extensive experience helping organizations adopt cloud strategies that maximize the value of their IT investments. 

As the Federal government embarks on a cloud technology modernization initiative, it has a major opportunity to improve the process by which departments and agencies transition to the cloud environment. As the draft strategy document notes, the 2010 strategy — “Cloud First” — provided agencies with broad authority to adopt cloud solutions. However, this strategy had several shortcomings; as a result, many agencies were slow to adapt. 

Although no two organizations have the same cloud needs, there are three major challenges that are common, to a greater or lesser degree, for all cloud transitions:

1.	Creating mission/business requirements that translate to cloud services
2.	Objectively evaluating vendor capabilities
3.	Implementing a cloud strategy

These challenges can be ameliorated by effective market research services.  

Market Research Helps Address Challenges  

Many organizations struggle with requirements definition. Though acquisition officials have become increasingly confident with defining requirements in certain areas of IT, cloud technology is relatively new to the public sector. As a result, the optimal set of requirements for cloud technology remains uncertain, difficult to define, and subject to misunderstandings and lack of clarity. Independent market research firms are well-equipped to assist government organizations with defining requirements, because they develop expertise in this area by repeatedly advising other organizations going through similar challenges. 

A second major challenge in acquiring cloud services and technologies is in the technical evaluation of vendor-provided data. Such data is inherently biased to favor the offeror’s solution, which makes objective analysis difficult. Furthermore, the volume and complexity of data that government organizations receive in response to a request for information or solicitation for a cloud solution can be overwhelming. Frequently, there is not sufficient time for a complete evaluation prior to reaching a major decision point, which can contribute to delays.

The third major challenge is actually putting a cloud strategy into execution, no matter how well planned or written the strategy may be.  Government agencies can have a disconnect between mission and IT organizations and research organizations such as Gartner specialize in connecting mission and technology leaders through academic research, clinics, workshops, and focused consulting engagements. Investments in cloud computing require an understanding and evaluation of business as well as technology factors, such as what type of workloads should be migrated to cloud, how to engage with cloud service providers, how to balance on-premise vs cloud workloads in “hybrid” scenarios, and how to manage stakeholder expectations.  

The Bottom Line

As a disinterested party, market research firms can act as an honest broker and sort through the bias present in vendor-provided data. These firms also retain a large breadth and depth of technical expertise that can quickly be brought to bear in response to an urgent government need. When properly utilized, independent market research firms increase speed and accuracy in the evaluation of cloud proposals. 

The Federal Cloud Computing Strategy should encourage acquisition officials to avail themselves of independent commercial market research services. Such services have the potential to help government organizations select the best technical solution for their cloud needs as well as increase the speed of adaptation.
